src/auth/screens/events.screen.js

```diff
@@ -206,103 +213,88 @@ class Events extends Component {
+            <LinkDescription
               onPress={() => this.navigateToRepository(userEvent)}
             >
               {userEvent.repo.name}
-            </Text>
+            </LinkDescription>
           );
         }
 
         return null;
       }
       case 'DeleteEvent':
         return (
-          <Text style={styles.deletedLinkBranchDescription}>
+          <DeletedLinkBranchDescription>
             {userEvent.payload.ref}
-          </Text>
+          </DeletedLinkBranchDescription>
         ); // can only be branch or tag
       case 'ForkEvent':
       case 'WatchEvent':
       case 'PublicEvent':
         return (
-          <Text
-            style={styles.linkDescription}
-            onPress={() => this.navigateToRepository(userEvent)}
-          >
+          <LinkDescription onPress={() => this.navigateToRepository(userEvent)}>
             {userEvent.repo.name}
-          </Text>
+          </LinkDescription>
         );
       case 'GollumEvent':
         return (
           <Text>
             the{' '}
-            <Text
-              style={styles.linkDescription}
+            <LinkDescription
               onPress={() => this.navigateToRepository(userEvent)}
             >
               {userEvent.repo.name}
-            </Text>{' '}
+            </LinkDescription>{' '}
             wiki
           </Text>
         );
       case 'IssueCommentEvent':
       case 'IssuesEvent':
         return (
-          <Text
-            style={styles.linkDescription}
-            onPress={() => this.navigateToIssue(userEvent)}
-          >
+          <LinkDescription onPress={() => this.navigateToIssue(userEvent)}>
             {userEvent.payload.issue.title}
-          </Text>
+          </LinkDescription>
         );
       case 'MemberEvent':
         return (
-          <Text
-            style={styles.linkDescription}
-            onPress={() => this.navigateToProfile(userEvent)}
-          >
+          <LinkDescription onPress={() => this.navigateToProfile(userEvent)}>
             {userEvent.payload.member.login}
-          </Text>
+          </LinkDescription>
         );
       case 'PullRequestEvent':
       case 'PullRequestReviewEvent':
       case 'PullRequestReviewCommentEvent':
         return (
-          <Text
-            style={styles.linkDescription}
-            onPress={() => this.navigateToIssue(userEvent)}
-          >
+          <LinkDescription onPress={() => this.navigateToIssue(userEvent)}>
             {userEvent.payload.pull_request.title}
-          </Text>
+          </LinkDescription>
         );
       case 'PushEvent':
         return (
-          <Text style={styles.linkDescription}>
+          <LinkBranchDescription>
             {userEvent.payload.ref.replace('refs/heads/', '')}
-          </Text>
+          </LinkBranchDescription>
         );
       case 'ReleaseEvent':
-        return `${userEvent.payload.release.id}`;
+        return `888 ${userEvent.payload.release.id}`;
```

Comment:
888? Seems like you mistyped something.

Reply:
Oops. Forgotten remnants of my pro debugging technique... 😅